Hydra/Serializer/CollectionNormalizer.php

```diff
@@ -92,7 +100,7 @@ public function normalize($object, $format = null, array $context = array())
                 '@context' => $this->contextBuilder->getResourceContext($resource, $context),
                 '@id' => $context['request_uri'],
             ];
-            list($parts, $parameters) = $this->parseRequestUri($context['request_uri']);
+            $uri = $this->parseRequestUri($context['request_uri']);
```

Comment:
I've just tried. The stable version (3.x) is useless for our purpose, and has already been declared EOL.